modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/database/H2Tree.java

```diff
@@ -59,6 +59,9 @@
  * H2 tree index implementation.
  */
 public class H2Tree extends BPlusTree<H2Row, H2Row> {
+    /** */
+    public static final String IGNITE_THROTTLE_INLINE_SIZE_CALCULATION = "IGNITE_THROTTLE_INLINE_SIZE_CALCULATION";
```

Comment:
in some case we didn`t ex: org.apache.ignite.internal.processors.cache.persistence.GridCacheDatabaseSharedManager#IGNITE_PDS_CHECKPOINT_TEST_SKIP_SYNC , i just don`t want to have a lot of changes instead of master code here, do u insist of this change ?

Reply:
If the property is not going to be used by user it can be kept here. Otherwise it is better to move it.